tests/test_dmlscheduler.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+import time
+import tests.context
+
+import logging
+import numpy as np
+from custom.keras               import get_optimizer
+from core.runner                import DMLRunner
+from core.runner_actor          import DMLRunnerActor
+from models.keras_perceptron    import KerasPerceptron
+from core.utils.dmljob          import DMLJob, serialize_job, deserialize_job
+from core.scheduler             import DMLScheduler
+from examples.labelers          import mnist_labeler
+from queue import Queue
+print("testing")
+
+
+import ray
+
+
+def make_dataset_path():
+    return 'datasets/mnist'
+
+
+def make_config():
```

Comment:
You can remove this function since you're not using it.

Reply:
Removed.